src/main/scala/io/sdkman/changelogs/GraceMigrations.scala

```diff
@@ -0,0 +1,59 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+import org.bson.Document
+
+@ChangeLog(order = "084")
+class GraceMigrations {
+
+  val CandidateName = "grace"
+
+  @ChangeSet(
+    order = "001",
+    id = "001_add_grace_candidate",
+    author = "rainboyan"
+  )
+  def migration001(implicit db: MongoDatabase) : Candidate = {
+    Candidate(
+      candidate = CandidateName,
+      name = "Grace Framework",
+      description =
+        "An open-source and powerful One-Person web application Framework to help developers quickly build Spring Boot applications written in the Groovy programing language. Grace is a forked of Grails 5 that started development in early 2022.",
```

Comment:
Can we change this copy a little?

```
Grace is a powerful open-source, One-Person web application Framework to help developers build Spring Boot applications rapidly using the Groovy programming language. Grace is a fork of Grails 5 that started development in early 2022.
```

Reply:
yeah, is it very long?
Delete the last sentence, what do you think?
Any others?

Reply:
It's updated now.